roles/docker/tasks/main.yml

```diff
@@ -36,3 +36,42 @@
   when:
     - l_use_crio
     - openshift_docker_is_node_or_master | bool
+
+- when:
+    - l_use_crio
+  block:
+    - name: stop the current running docker
+      systemd:
+        state: stopped
+        name: "{{ openshift.docker.service_name }}"
```

Comment:
This seems to be one of the roles that OPS is using in our config loop:

https://github.com/openshift/openshift-ansible-ops/blob/4b21d6423280e18aa7793d16d641b4d335ae0f0f/ops_tech_profiles/docker/3.7/meta/main.yml#L15

This change could lead to docker being stopped on every one of our config loop runs.

I know that right now "config" isn't supported by openshift-ansible.

Should OPS remove this from our config loop?

Or can we make this section idempotent?